app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/display/DisplayFragmentTest.kt

```diff
@@ -45,40 +45,40 @@ class DisplayFragmentTest {
         @BeforeClass
         fun setUpClass() {
             initFirebase()
-            Intents.init()
         }
 
         @JvmStatic
         @AfterClass
         fun tearDownClass() {
-            Intents.release()
             runBlocking{ emptyFirebaseStorage(Firebase.storage.reference) }
         }
     }
 
 
     @Before
     fun setUp() {
-        GeneralUtils.fillUserEntryAndEnterTheApp("client@gmail.com", "123456")
         context = InstrumentationRegistry.getInstrumentation().context
+        Intents.init()
     }
 
 
     @OptIn(ExperimentalCoroutinesApi::class)
     @After
     fun tearDown() = runTest {
         launch { emptyFirebaseStorage(FirebaseStorage.getInstance().reference) }.join()
+        Intents.release()
     }
 
     @OptIn(ExperimentalCoroutinesApi::class)
     @Test
     fun displayFragmentDisplayOnlyOnePhotoIfSame() = runTest {
```

Comment:
"displayFragment" as prefix doesn't add information about the test as it's already the test class name.
It should be removed.